### ISSUE TITLE
Standalone 1 NIC templates v4.3.0.0 - BIG-IP v13.0 VM network configuration is lost during deployment

### ISSUE THREAD

**Issue by edwoodjrjr** [F5Networks/f5-azure-arm-templates#47]
## Description
Due to a command required during BIG-IP onboarding, BIG-IP v13.0 VMs created by the 1 NIC ARM templates can lose their self IP and default route configuration during deployment.  This results in issues with DNS lookups and other network functionality after the BIG-IP VM is rebooted.

The workaround for this issue is to manually recreate the self IP and default route once the ARM deployment is complete.  This can be done by running the following commands from the shell or recreating the network configuration in the BIG-IP GUI:

```
tmsh create net self self_1nic address <management IP/network mask> vlan internal
tmsh create net route default gw <gateway for management subnet> 
```

For example:
```
tmsh create net self self_1nic address 10.0.0.4/24 vlan internal
tmsh create net route default gw 10.0.0.1
```

Note: The self IP address object MUST be named "self_1nic".

## Template
All 1 NIC templates.

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 3

Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.

**Comment by ianwijaya** [comment]
`
tmsh modify net self self_1nic allow-service default
`
must be configured, else arp will fail.

**Comment by jsevedge** [comment]
This issue has been resolved in the latest release, v4.4.0.0.
